<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPlacingPlan</class>
 <widget class="QWidget" name="EditPlacingPlan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>卡片放置方案编辑</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_top">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_general" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_general">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_file_path" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_file_path">
         <property name="spacing">
          <number>9</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_file_path">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>文件路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_file_path">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_file_path">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_common" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_common">
         <property name="spacing">
          <number>9</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBox_is_team_mode">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>组队模式</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_choose_section">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>选择方案：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_choose_section">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>120</horstretch>
             <verstretch>30</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_common">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_new_config">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>新建方案</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_modify_config">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>修改方案</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_delete_config">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>删除方案</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget_placing_config">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_1p_placing_config">
         <attribute name="title">
          <string>房主卡片放置方案</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_1p_placing_config" stretch="1,2,10">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_1p_setting" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_1p_setting">
             <property name="spacing">
              <number>9</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_1p_pos">
               <property name="minimumSize">
                <size>
                 <width>70</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>房主放置位置</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_1p_pos">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_1p_deck">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>选择卡组：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_1p_deck">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_1p_setting">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>576</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_1p_deck">
            <property name="flow">
             <enum>QListView::LeftToRight</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_1p_placing_table">
            <property name="title">
             <string>房主卡片放置信息</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_1p_placing_table">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTableWidget" name="tableWidget_1p_placing_table">
               <property name="editTriggers">
                <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <column>
                <property name="text">
                 <string>卡片名称</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>卡槽位置</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>放置位置</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>默认CD</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>操作</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2p_placing_config">
         <attribute name="title">
          <string>房客卡片放置方案</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2p_placing_config" stretch="0,1,5">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2p_setting" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2p_setting">
             <property name="spacing">
              <number>9</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2p_pos">
               <property name="minimumSize">
                <size>
                 <width>70</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>房客放置位置</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_2p_pos">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2p_deck">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>选择卡组：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2p_deck">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2p_setting">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>567</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_2p_deck">
            <property name="flow">
             <enum>QListView::LeftToRight</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2p_placing_table">
            <property name="title">
             <string>房客卡片放置信息</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2p_placing_table">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTableWidget" name="tableWidget_2p_placing_table">
               <property name="editTriggers">
                <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <column>
                <property name="text">
                 <string>卡片名称</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>卡槽位置</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>放置位置</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>默认CD</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>操作</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
